cvlr-log/src/core.rs

```diff
@@ -54,6 +57,10 @@ mod rt_impls {
     pub extern "C" fn CVT_calltrace_attach_location(_tag: &str, v: u64) {}
     #[no_mangle]
     pub extern "C" fn CVT_rule_location(_file: &str, _line: u64) {}
+    #[no_mangle]
+    pub extern "C" fn CVT_calltrace_scope_start(prefix: &str) {}
+    #[no_mangle]
+    pub extern "C" fn CVT_calltrace_scope_end(prefix: &str) {}
```

Comment:
Tiny nit - prefix should actually be called `scopeName`. (also is wrong in EVMVerifier)